packages/adapter-cloudflare-workers/index.js

```diff
@@ -11,9 +11,9 @@ import { fileURLToPath } from 'url';
 /**
  * @param {{
  *   esbuild?: (defaultOptions: BuildOptions) => Promise<BuildOptions> | BuildOptions;
- * }} options
+ * }} [options]
```

Comment:
Is this an intentional change? Why only on some adapters?

Reply:
- `adapter-cloudflare-workers`, `adapter-vercel`, `adapter-netlify`, and `adapter-node` all use esbuild, so this option should be available but not made mandatory.
- `adapter-begin` is unmaintained
- `adapter-static` doesn't use esbuild, so it wasn't affected.

If you're referring to the specific line, that's JSDoc's way of specifying an optional parameter so that you can do something like `adapter: adapterCfw()` without having to pass in an empty object.

Reply:
>If you're referring to the specific line, that's JSDoc's way of specifying an optional parameter so that you can do something like `adapter: adapterCfw()` without having to pass in an empty object.

I was referring to the specific line, thanks for clarifying